.github/workflows/ci.yml

```diff
@@ -0,0 +1,237 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - '*.x'
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - main
+  schedule:
+    # Weekly Monday 9AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 9 * * 1'
```

Comment:
If you are going to run this once a week I'd suggest maybe Sunday at 9am. There can be a lot of activity of the regression machines on Monday.

Reply:
This is a github workflow so it should be fine.

Reply:
No harm in changing it (and may be impactful in the future), so I did so.